src/main/steps/applicant2/address/confirm-change/template.njk

```diff
@@ -0,0 +1,8 @@
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
```

Comment:
If form fields are used then this can probably be achieved without using template.njk (Only applicable if page is not visually different as per prototype)